src/thor/multimodal.cc

```diff
@@ -42,9 +42,10 @@ namespace thor {
 MultiModalPathAlgorithm::MultiModalPathAlgorithm(const boost::property_tree::ptree& config)
     : PathAlgorithm(config.get<uint32_t>("max_reserved_labels_count_astar",
                                          kInitialEdgeLabelCountAstar),
-                    config.get<bool>("clear_reserved_memory", false)),
-      max_walking_dist_(0), max_label_count_(std::numeric_limits<uint32_t>::max()),
-      mode_(travel_mode_t::kPedestrian), travel_type_(0) {
+                    config.get<bool>("clear_reserved_memory", false)) {
+  mode_ = travel_mode_t::kPedestrian;
+  travel_type_ = 0;
+  max_walking_dist_ = 0;
```

Comment:
why move them out of the initializer list? if it was a warning, just reorder them to shut the warning up

Reply:
That was more of a preference call, but I can revert. I find it more legible. My eyes have a hard time seeing all variables in long initializer lists